README.md

```diff
@@ -68,3 +68,23 @@ Shutdown Instructions (Optional)
 3. Access your zookeeper instance: `/PATH/TO/zookeeper/bin/zkCli.sh`
 4. Remove hdfs-mesos framework state from zookeeper: `rmr /hdfs-mesos`
 5. (Optional) Clear your data directories as specified in your `mesos-site.xml`. This is necessary to relaunch HDFS in the same directory.
+
+Docker Usage (Optional)
+--------------------------
+
+### Use [Official Mesosphere Image](https://registry.hub.docker.com/u/mesosphere/hdfs-mesos/)
```

Comment:
I just mean that when I go to https://registry.hub.docker.com/u/mesosphere/hdfs-mesos/ there is absolutely nothing under the "Information" tab. Users at least like to see a description of what the project is, and a link to the github repo hosting the Dockerfile, so they can know what they're pulling down, rather than randomly trusting that you're not installing malware/bloatware for them.
Would also be nice to get this into an automated build&publish setup, so it's clear that what's in Dockerhub is guaranteed to be the latest stable build of hdfs-mesos.